src/metrics.ts

```diff
@@ -90,13 +92,18 @@ export default class Metrics extends EventEmitter {
     this.httpOptions = httpOptions;
   }
 
+  private getAppliedJitter() {
+    const jitter = Math.random() * this.metricsJitter;
+    return Math.random() < 0.5 ? -jitter : jitter;
+  }
```

Comment:
This could be broken out as a unit and tested with a unit test. That would at least be a way to test this functionality in a simple way. Would you mind performing this change?